### ISSUE TITLE
Save whether the terminal panel is zoomed

### ISSUE THREAD

**Issue by Liamolucko** [zed-industries/community#1614]
### Check for existing issues

- [X] Completed

### Describe the feature

I was a big fan of using the dock in 'expanded' mode before it was removed. I can get 95% of the way towards the old functionality by zooming the terminal panel, after which point I can press `` ctrl+` `` to toggle the terminal at will.

However, it's a bit irritating that every time I reopen Zed/open a new workspace I have to manually open the terminal, put it into zoomed mode, then close it again to get things working the way I want. It'd be nice if Zed would automatically save whether the terminal panel is zoomed, in the same way that it saves the size of the terminal panel (including the fact that it's the same across all workspaces, not per-workspace).

This is a pretty small papercut, it only takes a second to zoom the terminal anyway, but it'd still be nice if it were fixed.

**Comment by JosephTLyons** [comment]
Yes, the is something we discussed and I think we should definitely serialize the setting to the workspace per project.

**Comment by Liamolucko** [comment]
> I think we should definitely serialize the setting to the workspace per project

I'd personally prefer the opposite, having it saved globally, since I want it to stay zoomed regardless of what I'm working on. I'd still be fine with either option though, since it's not _that_ often that I open a new project anyway.
